generate_endf.py

```diff
@@ -137,14 +145,77 @@ def sort_key(path):
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinC5O2H8.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinH2O.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinCH2.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinZrH.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinIceIh.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinYH2.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-ortho-H.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-para-H.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-benzene.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-l-CH4.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-s-CH4.endf'),
+                (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-DinD2O.endf'),
+                (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-ortho-D.endf'),
+                (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-para-D.endf'),
+                (neutron_dir / 'n-004_Be_009.endf', thermal_dir / 'tsl-BeinBeO.endf'),
+                (neutron_dir / 'n-004_Be_009.endf', thermal_dir / 'tsl-Be-metal.endf'),
+                (neutron_dir / 'n-006_C_012.endf', thermal_dir / 'tsl-CinSiC.endf'),
+                (neutron_dir / 'n-006_C_012.endf', thermal_dir / 'tsl-crystalline-graphite.endf'),
+                (neutron_dir / 'n-006_C_012.endf', thermal_dir / 'tsl-reactor-graphite-10P.endf'),
+                (neutron_dir / 'n-006_C_012.endf', thermal_dir / 'tsl-reactor-graphite-30P.endf'),
+                (neutron_dir / 'n-007_N_014.endf', thermal_dir / 'tsl-NinUN.endf'),
+                (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinBeO.endf'),
+                (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinD2O.endf'),
+                (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinIceIh.endf'),
+                (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinUO2.endf'),
+                (neutron_dir / 'n-013_Al_027.endf', thermal_dir / 'tsl-013_Al_027.endf'),
+                (neutron_dir / 'n-026_Fe_056.endf', thermal_dir / 'tsl-026_Fe_056.endf'),
+                (neutron_dir / 'n-014_Si_028.endf', thermal_dir / 'tsl-SiinSiC.endf'),
+                (neutron_dir / 'n-014_Si_028.endf', thermal_dir / 'tsl-SiO2-alpha.endf'),
+                (neutron_dir / 'n-014_Si_028.endf', thermal_dir / 'tsl-SiO2-beta.endf'),
+                (neutron_dir / 'n-039_Y_089.endf', thermal_dir / 'tsl-YinYH2.endf'),
+                (neutron_dir / 'n-040_Zr_090.endf', thermal_dir / 'tsl-ZrinZrH.endf'),
+                (neutron_dir / 'n-092_U_238.endf', thermal_dir / 'tsl-UinUN.endf'),
+                (neutron_dir / 'n-092_U_238.endf', thermal_dir / 'tsl-UinUO2.endf')
```

Comment:
Looks like `thermal_dir` here should be `neutron_dir` because the thermal scattering ENDF files get extracted to the neutron directory (same goes for the sab_files list for b7.1)

Reply:
thanks Paul, I've corrected the dir